### ISSUE TITLE
Clean up docs/core/howto/defer.rst

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8331]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8331|
|Type|task|
|Created|2016-05-16 00:39:17Z|
|Branch|https://github.com/twisted/twisted/tree/defer-docs-clean-8331|

Should meet our docs standards and be otherwise editable.

<details><summary>Searchable metadata</summary>

```
trac-id__8331 8331
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__defer_docs_clean_8331 defer-docs-clean-8331
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1463359157225804 1463359157225804
changetime__1463415722428970 1463415722428970
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Docs builder is looking good, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good... assuming that this cleanup has no content changes, but just re-wrapping the lines.

thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 95acb5c5cbd61cc3e8b780579f1c36b72735d515
```
#!CommitTicketReference repository="" revision="95acb5c5cbd61cc3e8b780579f1c36b72735d515"
Merge defer-docs-clean-8331: Clean up docs/core/howto/defer.rst

Author: hawkowl
Reviewer: adiroiban
Fixes: #8331
```
